In [377]:
import numpy as np
import pandas as pd
import matplotlib.pyplot as plt
import seaborn as sns

In [378]:
df=pd.read_csv('WELLCOME_APCspend2013_forThinkful.csv',encoding='latin1')

In [379]:
df.head()

PMID/PMCID Publisher           Journal title  \
0                    NaN       CUP  Psychological Medicine   
1             PMC3679557       ACS       Biomacromolecules   
2  23043264  PMC3506128        ACS              J Med Chem   
3    23438330 PMC3646402       ACS              J Med Chem   
4   23438216 PMC3601604        ACS              J Org Chem   

                                       Article title  \
0  Reduced parahippocampal cortical thickness in ...   
1  Structural characterization of a Model Gram-ne...   
2  Fumaroylamino-4,5-epoxymorphinans and related ...   
3  Orvinols with mixed kappa/mu opioid receptor a...   
4  Regioselective opening of myo-inositol orthoes...   

  COST (£) charged to Wellcome (inc VAT when charged)  
0                                              £0.00   
1                                           £2381.04   
2                                            £642.56   
3                                            £669.64   
4                                            £685.88

In [380]:
#First we will look at the publisher column to consolidate similar publisher names.

#Stripping white space
df['Publisher'] = df['Publisher'].str.strip()
#Capitalizing each publisher
df['Publisher'] = df['Publisher'].str.lower()
df['Publisher'] = df['Publisher'].str.capitalize()
#Checking Unique Publisher Titles
df['Publisher'].unique()
len(df['Publisher'].unique())

254

In [381]:

#Begin consolidating publisher names
#1
df['Publisher'] = df['Publisher'].str.replace('Acs','American chemical society')
df['Publisher'] = df['Publisher'].str.replace('American chemical society publications','American chemical society')
df['Publisher'] = df['Publisher'].str.replace('Acs (amercian chemical society) publications','American chemical society')
df['Publisher'] = df['Publisher'].str.replace('American chemical society publications','American chemical society')
df['Publisher'] = df['Publisher'].str.replace('Acs publications','American chemical society')
df['Publisher'] = df['Publisher'].str.replace('American chemical society (amercian chemical society) publications','American chemical society')
df['Publisher'] = df['Publisher'].str.replace('The american chemical society petroleum research trust','American chemical society')


#2
df['Publisher'] = df['Publisher'].str.replace('American soc for biochemistry and molecular biology','American society for biochemistry and molecular biology')
df['Publisher'] = df['Publisher'].str.replace('American society for biochemistry and molecular biolgy','American society for biochemistry and molecular biology')
df['Publisher'] = df['Publisher'].str.replace('Asbmb','American society for biochemistry and molecular biology')
df['Publisher'] = df['Publisher'].str.replace('Asbmb cadmus','American society for biochemistry and molecular biology')
df['Publisher'] = df['Publisher'].str.replace('American society for biochemistry and molecular biology cadmus','American society for biochemistry and molecular biology')
df['Publisher'] = df['Publisher'].str.replace('Ambsb','American society for biochemistry and molecular biology')
df['Publisher'] = df['Publisher'].str.replace('Asbmb/cadmus','American society for biochemistry and molecular biology')
df['Publisher'] = df['Publisher'].str.replace('American society for biochemistry and molecular biology/cadmus','American society for biochemistry and molecular biology')

#3.
df['Publisher'] = df['Publisher'].str.replace('American society of microbiology','American society for microbiology')
df['Publisher'] = df['Publisher'].str.replace('Asm (american society for microbiology)','American society for microbiology')
df['Publisher'] = df['Publisher'].str.replace('Asbmc /cenveo','American society for microbiology')
df['Publisher'] = df['Publisher'].str.replace('Asm (american society for microbiology)','American society for microbiology')
df['Publisher'] = df['Publisher'].str.replace('Asm','American society for microbiology')
df['Publisher'] = df['Publisher'].str.replace('American society for microbiology (american society for microbiology)','American society for microbiology')
df['Publisher'] = df['Publisher'].str.replace('Society for general microbiology','American society for microbiology')
df['Publisher'] = df['Publisher'].str.replace('Society for genermal microbiology','American society for microbiology')
df['Publisher'] = df['Publisher'].str.replace('Society of general microbiology','American society for microbiology')

#4.
df['Publisher'] = df['Publisher'].str.replace('American society for biochemistry and molecular biology/cenveo publisher services','American society for biochemistry and molecular biology')
df['Publisher'] = df['Publisher'].str.replace('American society for biochemistry and molecular biology/cenveo publisher services','American society for biochemistry and molecular biology')
df['Publisher'] = df['Publisher'].str.replace('The american society for biochemistry and molecular biology','American society for biochemistry and molecular biology')
df['Publisher'] = df['Publisher'].str.replace('The american society for biochemistry and molecular biology, inc','American society for biochemistry and molecular biology')
df['Publisher'] = df['Publisher'].str.replace('American society for biochemistry and molecular biology, inc','American society for biochemistry and molecular biology')


#5.
df['Publisher'] = df['Publisher'].str.replace('American society of hamatology', 'American society of hematology')
df['Publisher'] = df['Publisher'].str.replace('American society of haematology', 'American society of hematology')
#6
df['Publisher'] = df['Publisher'].str.replace('Benthan science publishers', 'Bentham science publishers')

#7
df['Publisher'] = df['Publisher'].str.replace('Benthan science publishers', 'Bentham science publishers')

#8
df['Publisher'] = df['Publisher'].str.replace('Biomed central limited', 'Biomed central')
df['Publisher'] = df['Publisher'].str.replace('Biomed central ltd', 'Biomed central')
df['Publisher'] = df['Publisher'].str.replace('Bmc', 'Biomed central')


#9
df['Publisher'] = df['Publisher'].str.replace('Bmj', 'British medical journal')
df['Publisher'] = df['Publisher'].str.replace('Bmj journals', 'British medical journal')
df['Publisher'] = df['Publisher'].str.replace('Bmj publishing group ltd & british thoracic society', 'British medical journal')
df['Publisher'] = df['Publisher'].str.replace('Bmj publishing group', 'British medical journal')
df['Publisher'] = df['Publisher'].str.replace('Bmj publishing group ltd', 'British medical journal')
df['Publisher'] = df['Publisher'].str.replace('British medical journal group', 'British medical journal')
df['Publisher'] = df['Publisher'].str.replace('British medical journal journals', 'British medical journal')
df['Publisher'] = df['Publisher'].str.replace('British medical journal publishing group', 'British medical journal')
df['Publisher'] = df['Publisher'].str.replace('British medical journal publishing group ltd', 'British medical journal')
df['Publisher'] = df['Publisher'].str.replace('British medical journal ltd & british thoracic society', 'British medical journal')
df['Publisher'] = df['Publisher'].str.replace('British medical journal ltd', 'British medical journal')

#10
df['Publisher'] = df['Publisher'].str.replace('Cadmus', 'Cadmus journal services')
df['Publisher'] = df['Publisher'].str.replace('Cadmus', 'Cadmus journal services')
df['Publisher'] = df['Publisher'].str.replace('Cadmus journal service', 'Cadmus journal services')
df['Publisher'] = df['Publisher'].str.replace('Cadmus journal servicess journal services', 'Cadmus journal services')
df['Publisher'] = df['Publisher'].str.replace('Cadmus journal services journal service', 'Cadmus journal services')
df['Publisher'] = df['Publisher'].str.replace('Cadmus journal servicess jouranl services journal services', 'Cadmus journal services')
df['Publisher'] = df['Publisher'].str.replace('Cadmus journal services journal services', 'Cadmus journal services')
df['Publisher'] = df['Publisher'].str.replace('Cadmus journal servicess', 'Cadmus journal services')
df['Publisher'] = df['Publisher'].str.replace('Camdus journal services', 'Cadmus journal services')

#11
df['Publisher'] = df['Publisher'].str.replace('Cambridge univ press', 'Cambridge university press')
df['Publisher'] = df['Publisher'].str.replace('Cambridge uni press', 'Cambridge university press')
df['Publisher'] = df['Publisher'].str.replace('Cambridge journals', 'Cambridge university press')
df['Publisher'] = df['Publisher'].str.replace('Cup', 'Cambridge university press')


#12
df['Publisher'] = df['Publisher'].str.replace('Cold spring harbor laboratory press', 'Cold spring harbor')
df['Publisher'] = df['Publisher'].str.replace('Cold spring habour press', 'Cold spring harbor')
df['Publisher'] = df['Publisher'].str.replace('Cold spring harbor press', 'Cold spring harbor')
df['Publisher'] = df['Publisher'].str.replace('Cold spring harbor publications', 'Cold spring harbor')
df['Publisher'] = df['Publisher'].str.replace('Cshlp', 'Cold spring harbor')

#13
df_pub = df_pub.str.replace('Cenveo publisher services/asm jv1', 'Cenveo publisher services')

#14
df['Publisher'] = df['Publisher'].str.replace('Company of biologist', 'Company of biologists')
df['Publisher'] = df['Publisher'].str.replace('Company of biologistss ltd', 'Company of biologists')
df['Publisher'] = df['Publisher'].str.replace('Company of biologistss', 'Company of biologists')


#15
df['Publisher'] = df['Publisher'].str.replace('Darmouth journal services', 'Dartmouth journal services')
df['Publisher'] = df['Publisher'].str.replace('Dartmouth journals', 'Dartmouth journal services')

#16
df['Publisher'] = df['Publisher'].str.replace('Elseveier science', 'Elsevier')
df['Publisher'] = df['Publisher'].str.replace('Elsevier / cell science', 'Elsevier')
df['Publisher'] = df['Publisher'].str.replace('Elsevier ltd', 'Elsevier')
df['Publisher'] = df['Publisher'].str.replace('Elsevier b.v.', 'Elsevier')
df['Publisher'] = df['Publisher'].str.replace('Elsevier/cell press', 'Elsevier')
df['Publisher'] = df['Publisher'].str.replace('Elsevier (cell press)', 'Elsevier')

#17
df['Publisher'] = df['Publisher'].str.replace('Endocrine society', 'European society of endocrinology')
df['Publisher'] = df['Publisher'].str.replace('European society of endocrinolog', 'European society of endocrinology')
df['Publisher'] = df['Publisher'].str.replace('Publisher society for endocrinology', 'European society of endocrinology')
df['Publisher'] = df['Publisher'].str.replace('Society for endocrinology','European society of endocrinology')
df['Publisher'] = df['Publisher'].str.replace('The endocrine society', 'European society of endocrinology')
df['Publisher'] = df['Publisher'].str.replace('Endocrine socety', 'European society of endocrinology')
df['Publisher'] = df['Publisher'].str.replace('The endrocrine society', 'European society of endocrinology')

#18

df['Publisher'] = df['Publisher'].str.replace('Faseb', 'Federation of american societies for experimental biology')
df['Publisher'] = df['Publisher'].str.replace('Federation of american societies for experimental biology (faseb)','Federation of american societies for experimental biology')
df['Publisher'] = df['Publisher'].str.replace('Federation of the american society of experimental biology','Federation of american societies for experimental biology')

#19
df['Publisher'] = df['Publisher'].str.replace('Frontiers media sa', 'Frontiers research foundation')
df['Publisher'] = df['Publisher'].str.replace('Frontiers media', 'Frontiers research foundation')
df['Publisher'] = df['Publisher'].str.replace('Frontiers', 'Frontiers research foundation')
df['Publisher'] = df['Publisher'].str.replace('Frontiers Research Foundation Research Foundation', 'Frontiers research foundation')
df['Publisher'] = df['Publisher'].str.replace('Frontiers research foundation research foundation', 'Frontiers research foundation')

#20
df['Publisher'] = df['Publisher'].str.replace('Future medicine', 'Future science')
df['Publisher'] = df['Publisher'].str.replace('Future medicine ltd', 'Future science')

#21
df['Publisher'] = df['Publisher'].str.replace('Hindawi', 'Hindawi publishing corporation')
df['Publisher'] = df['Publisher'].str.replace('Hindawi publishing corporation publishing corporation', 'Hindawi publishing corporation')

#22
df['Publisher'] = df['Publisher'].str.replace('Impact journals llc', 'Impact journals')

#23
df['Publisher'] = df['Publisher'].str.replace('Informa healthcare communications', 'Informa healthcare')

#24
df['Publisher'] = df['Publisher'].str.replace('International union of crystallography (iucr)', 'International union of crystallography')

#25
df['Publisher'] = df['Publisher'].str.replace('Iop publishing', 'Institute of physics')

#26
df['Publisher'] = df['Publisher'].str.replace('John wiley', 'John wiley and sons')
df['Publisher'] = df['Publisher'].str.replace('John wiley & sons', 'John wiley and sons')
df['Publisher'] = df['Publisher'].str.replace('john wiley & sons', 'John wiley and sons')
df['Publisher'] = df['Publisher'].str.replace('John wiley & sons inc', 'John wiley and sons')
df['Publisher'] = df['Publisher'].str.replace('John wiley & sons, inc.', 'John wiley and sons')
df['Publisher'] = df['Publisher'].str.replace('John wiley & sons ltd', 'John wiley and sons')
df['Publisher'] = df['Publisher'].str.replace('John wiley and sons ltd', 'John wiley and sons')
df['Publisher'] = df['Publisher'].str.replace('John wiley', 'John wiley and sons')

#27
df['Publisher'] = df['Publisher'].str.replace('Jove', 'Journal of visualized experiments')
df['Publisher'] = df['Publisher'].str.replace('My jove corp', 'Journal of visualized experiments')
df['Publisher'] = df['Publisher'].str.replace('My jove corporation', 'Journal of visualized experiments')
df['Publisher'] = df['Publisher'].str.replace('Myjove corporation', 'Journal of visualized experiments')

#28
df['Publisher'] = df['Publisher'].str.replace('Landes biosciences', 'Landes bioscience')

#29
df['Publisher'] = df['Publisher'].str.replace('Mary ann liebert, inc.', 'Mary ann liebert')
df['Publisher'] = df['Publisher'].str.replace('Mary ann liebert inc', 'Mary ann liebert')

#30
df['Publisher'] = df['Publisher'].str.replace('Mary ann liebert inc', 'Nat')

#31
df['Publisher'] = df['Publisher'].str.replace('National academy of sciences of the united states of america', 'National academy of sciences usa')
df['Publisher'] = df['Publisher'].str.replace('National academy of sciences, usa', 'National academy of sciences usa')
df['Publisher'] = df['Publisher'].str.replace('National academy of sciences', 'National academy of sciences usa')
df['Publisher'] = df['Publisher'].str.replace('National academy of sciences usa usa', 'National academy of sciences usa')
df['Publisher'] = df['Publisher'].str.replace('Pnas author publication', 'National academy of sciences usa')
df['Publisher'] = df['Publisher'].str.replace('Pnas', 'National academy of sciences usa')
df['Publisher'] = df['Publisher'].str.replace('Proceedings of the national academy of sciences (pnas)', 'National academy of sciences usa')


#32
df['Publisher'] = df['Publisher'].str.replace('Nature publishing', 'Nature')
df['Publisher'] = df['Publisher'].str.replace('Nature pg', 'Nature')
df['Publisher'] = df['Publisher'].str.replace('Nature publishing group', 'Nature')
df['Publisher'] = df['Publisher'].str.replace('Nature group ltd', 'Nature')
df['Publisher'] = df['Publisher'].str.replace('Nature group', 'Nature')
df['Publisher'] = df['Publisher'].str.replace('Npg', 'Nature')

#33
df['Publisher'] = df['Publisher'].str.replace('Oup', 'Oxford university press')
df['Publisher'] = df['Publisher'].str.replace('Oxford journals', 'Oxford university press')
df['Publisher'] = df['Publisher'].str.replace('Oxford univ press', 'Oxford university press')
df['Publisher'] = df['Publisher'].str.replace('Oxford university press (oup)', 'Oxford university press')
df['Publisher'] = df['Publisher'].str.replace('Oxford journals (oup)', 'Oxford university press')

#34
df['Publisher'] = df['Publisher'].str.replace('Plos (public library of science)', 'Plos public library of science')
df['Publisher'] = df['Publisher'].str.replace('Plos', 'Plos public library of science')

#35
df['Publisher'] = df['Publisher'].str.replace('Portland press ltd', 'Portland press')

#36
df['Publisher'] = df['Publisher'].str.replace('Pubmed', 'Pubmed central')

#37
df['Publisher'] = df['Publisher'].str.replace('Royal society', 'Royal society for chemistry')
df['Publisher'] = df['Publisher'].str.replace('Rsc', 'Royal society for chemistry')
df['Publisher'] = df['Publisher'].str.replace('Rsc publishing', 'Royal society for chemistry')
df['Publisher'] = df['Publisher'].str.replace('Royal society of chemistry', 'Royal society for chemistry')

#38
df['Publisher'] = df['Publisher'].str.replace('Sage publications inc', 'Sage')
df['Publisher'] = df['Publisher'].str.replace('Sage publishers', 'Sage')
df['Publisher'] = df['Publisher'].str.replace('Sage publications ltd', 'Sage')
df['Publisher'] = df['Publisher'].str.replace('Sage publishing', 'Sage')
df['Publisher'] = df['Publisher'].str.replace('Sage publications', 'Sage')

#39
df['Publisher'] = df['Publisher'].str.replace('Society for neurosciences', 'Society for neuroscience')
df['Publisher'] = df['Publisher'].str.replace('Society of neuro sciences', 'Society for neuroscience')
df['Publisher'] = df['Publisher'].str.replace('Society of neuroscience', 'Society for neuroscience')

#40
df['Publisher'] = df['Publisher'].str.replace('Springer - verlag gmbh', 'Springer')
df['Publisher'] = df['Publisher'].str.replace('Springer science + business media', 'Springer')
df['Publisher'] = df['Publisher'].str.replace('Springer verlag', 'Springer')
df['Publisher'] = df['Publisher'].str.replace('Springer-verlag gmbh', 'Springer')
df['Publisher'] = df['Publisher'].str.replace('Springer-verlag gmbh & ci', 'Springer')
df['Publisher'] = df['Publisher'].str.replace('Springer-veriag gmbh', 'Springer')
df['Publisher'] = df['Publisher'].str.replace('Springer & ci', 'Springer')
df['Publisher'] = df['Publisher'].str.replace('Springer science + business media', 'Springer')
df['Publisher'] = df['Publisher'].str.replace('Springer, heidelberger platz 3, d-14197 berlin', 'Springer')

#41
df['Publisher'] = df['Publisher'].str.replace('Taylor & francis', 'Taylor and francis')
df['Publisher'] = df['Publisher'].str.replace('Taylor & francis journals', 'Taylor and francis')
df['Publisher'] = df['Publisher'].str.replace('T&f', 'Taylor and francis')
df['Publisher'] = df['Publisher'].str.replace('Taylor and francis journals', 'Taylor and francis')

#42
df['Publisher'] = df['Publisher'].str.replace('The company of biolgists', 'The company of biologists')
df['Publisher'] = df['Publisher'].str.replace('The company of biologists ltd', 'The company of biologists')

#42
df['Publisher'] = df['Publisher'].str.replace('The company of biologists ltd', 'The company of biologists')
df['Publisher'] = df['Publisher'].str.replace('The company of biologists ltd', 'The company of biologists')

#43
df['Publisher'] = df['Publisher'].str.replace('Wiley subscription services', 'Wiley-blackwell')
df['Publisher'] = df['Publisher'].str.replace('Wiley subscription services inc.', 'Wiley-blackwell')
df['Publisher'] = df['Publisher'].str.replace('Wiley/blackwell', 'Wiley-blackwell')
df['Publisher'] = df['Publisher'].str.replace('Wiley vch', 'Wiley-blackwell')
df['Publisher'] = df['Publisher'].str.replace('Wiley subscription services inc', 'Wiley-blackwell')
df['Publisher'] = df['Publisher'].str.replace('Wiley subscription serviices inc', 'Wiley-blackwell')
df['Publisher'] = df['Publisher'].str.replace('Wiley subscription services', 'Wiley-blackwell')
df['Publisher'] = df['Publisher'].str.replace('Wiley blackwell', 'Wiley-blackwell')
df['Publisher'] = df['Publisher'].str.replace('Wiley', 'Wiley-blackwell')
df['Publisher'] = df['Publisher'].str.replace('Wiley-vch', 'Wiley-blackwell')
df['Publisher'] = df['Publisher'].str.replace('Wiley subscription services', 'Wiley-blackwell')
df['Publisher'] = df['Publisher'].str.replace('Wliey-blackwell', 'Wiley-blackwell')

#44
df['Publisher'] = df['Publisher'].str.replace('Wolters kluwer n.v./lippinott', 'Wolters kluwer')
df['Publisher'] = df['Publisher'].str.replace('Wolters kluwer health', 'Wolters kluwer')
df['Publisher'] = df['Publisher'].str.replace('Wolters kluwers', 'Wolters kluwer')
df['Publisher'] = df['Publisher'].str.replace('Wolters kluwer', 'Wolters kluwer')


df['Publisher'].unique()



 
     


array(['Cambridge university press', 'American chemical society',
       'American chemical society (amercian chemical society) publications',
       'Aga institute',
       'American society for biochemistry and molecular biology',
       'American association of immunologists',
       'American college of chest physicians',
       'American physiological society',
       'American psychiatric association',
       'American psychiatric publishing',
       'American psychological association',
       'American public health association',
       'American society for investigative pathology',
       'American society for microbiology',
       'American society for nutrition', 'American society of hematology',
       'American society of human genetics (elsevier)',
       'American speech-language-hearing association',
       'American society for microbiology (american society for microbiology)',
       'Association for research in vision & ophthalmology',
       'Bentham science publis

In [382]:
len(df['Publisher'].unique())

139

We have 139 unique Publisher names in our data set. 

In [383]:
df['Publisher'].value_counts().head()

Elsevier                     406
Public library of science    278
Oxford university press      161
Wiley-blackwell              149
Biomed central                95
Name: Publisher, dtype: int64

Elsevier is the most common  Journal with 406 articles. It is followed by Public library of science, Oxford university press, Wiley-blackwell, and Biomed central.

In [384]:
#Remove the money sign in the cost collumn and convert to float
df['COST (£) charged to Wellcome (inc VAT when charged)'] = df['COST (£) charged to Wellcome (inc VAT when charged)'].str.replace('£', '')
df['COST (£) charged to Wellcome (inc VAT when charged)'] = df['COST (£) charged to Wellcome (inc VAT when charged)'].str.replace('$', '')
df['COST (£) charged to Wellcome (inc VAT when charged)'] = df['COST (£) charged to Wellcome (inc VAT when charged)'].astype(float)




In [389]:
#Calculate statistics on top 5 journals
df_el = df.loc[df['Publisher'] == 'Elsevier', 'COST (£) charged to Wellcome (inc VAT when charged)']
df_plos = df.loc[df['Publisher'] == 'Public library of science', 'COST (£) charged to Wellcome (inc VAT when charged)']
df_oup = df.loc[df['Publisher'] == 'Oxford university press', 'COST (£) charged to Wellcome (inc VAT when charged)']
df_wb = df.loc[df['Publisher'] == 'Wiley-blackwell', 'COST (£) charged to Wellcome (inc VAT when charged)']
df_bc = df.loc[df['Publisher'] == 'Biomed central', 'COST (£) charged to Wellcome (inc VAT when charged)']
print('The mean cost of Elsevier journals is {}.'.format(df_el.mean()))
print('The median cost of Elsevier journals is {}.'.format(df_el.median()))
print('The standard deviation of the cost of Elsevier journals is {}.'.format(df_el.std()))
print('The mean cost of Public library of science journals is {}.'.format(df_plos.mean()))
print('The median cost of Public library of science journals is {}.'.format(df_plos.median()))
print('The standard deviation of the cost of Public library of science journals is {}.'.format(df_plos.std()))
print('The mean cost of Oxford university press journals is {}.'.format(df_oup.mean()))
print('The median cost of Oxford university press journals is {}.'.format(df_oup.median()))
print('The standard deviation of the cost of Oxford university press journals is {}.'.format(df_oup.std()))
print('The mean cost of Wiley-blackwell journals is {}.'.format(df_wb.mean()))
print('The median cost of Wiley-blackwell journals is {}.'.format(df_wb.median()))
print('The standard deviation of the cost of Wiley-blackwell journals is {}.'.format(df_wb.std()))
print('The mean cost of Biomed central journals is {}.'.format(df_bc.mean()))
print('The median cost of Biomed central journals is {}.'.format(df_bc.median()))
print('The standard deviation of the cost of Biomed central journals is {}.'.format(df_bc.std()))


The mean cost of Elsevier journals is 26998.890640394082.
The median cost of Elsevier journals is 2351.795.
The standard deviation of the cost of Elsevier journals is 154812.6007433407.
The mean cost of Public library of science journals is 55724.13561151078.
The median cost of Public library of science journals is 1040.455.
The standard deviation of the cost of Public library of science journals is 226208.96548803797.
The mean cost of Oxford university press journals is 20438.357080745343.
The median cost of Oxford university press journals is 2040.0.
The standard deviation of the cost of Oxford university press journals is 135400.26255899222.
The mean cost of Wiley-blackwell journals is 23371.878456375856.
The median cost of Wiley-blackwell journals is 1974.72.
The standard deviation of the cost of Wiley-blackwell journals is 141411.36313997747.
The mean cost of Biomed central journals is 22364.36873684211.
The median cost of Biomed central journals is 1315.8.
The standard deviation 